web/components/admin/ApplicantList/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/* eslint-disable camelcase */
+
+class ApplicantList extends React.Component {
+  render() {
+    const { applicants, selectedApplicantId, onApplicantClick, className } = this.props;
+    let applicantNumber = 0;
```

Comment:
hm not sure about the purpose of `applicantNumber`. Would it be more helpful to display the id instead?

Reply:
We used the applicant number last year as a low-effort way to assign applicants to organizers (ex: "`________, can you mark resumes for applicants 100-200`"). 

If it helps we can show the applicant ID in the full info instead. Last year we also had custom URLs for applicants (`admin/assessment/<applicant-id>` to automatically jump to the applicant via a URL). But that can come later if we have time. Is there some other reason you want `id` displayed?

Reply:
i'm just confused because applicant 1 is the first applicant displayed and the same applicant 1 can be different based on filters right?

Reply:
That's true, I can't tell from the mocks if that's the intended behavior nor do I remember if that's how it worked last year. But the current functionality might be useful if you want to say `"_______, can you mark resumes for UW applicants 100-200"` or something like that. I'll ask on figma